Core/bin/google-cloud-batch

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Core\Batch;
+
+use Google\Cloud\Core\Daemon\DaemonUtils;
```

Comment:
How will this script know how to find this class? (same for the debugger daemon).

This script is invoked directly like `vendor/bin/google-cloud-batch daemon`

Reply:
That is an excellent point, and seems fatal to this approach. Is there any workaround? I can't think of one, so I may just need to revert this.

Reply:
Agh, yeah. We would just end up with the same problem again trying to find and require the helper script. Sorry for taking things off course!

Reply:
Fixed